src/mappings/nodes.ts

```diff
@@ -15,6 +15,9 @@ import { Ctx } from '../processor'
 import assert from "assert";
 import { allowedNodeEnvironmentFlags } from "process";
 
+const ZOSUsedSRU = 107374182400;
+const ZOSMinUsedMemory = 2147483648;
```

Comment:
What are these numbers and where it came from?
Please add a code comment explaining these magic numbers.